consent-ws/src/test/resources/json/consent-group-name-duplicated.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "consentId": "testId3",
+    "groupName": "Vestibulum sed / 33-44"
+  },
+  {
+    "consentId": "testId3",
+    "groupName": "wrong group name"
+  },
+  {
+    "consentId": "",
+    "groupName": "empty consent Id"
+  }
+]
```

Comment:
This test case doesn't actually have duplicated group names. It does have duplicated consent IDs, so I think you might need to add a separate test for the duplicate group name case.